func main() {
            panic("Something went wrong");
        }